<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskComposerM</class>
 <widget class="QMainWindow" name="TaskComposerM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cchelper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="helpButton">
        <property name="text">
         <string>H</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="settingButton">
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="solveButton">
        <property name="font">
         <font>
          <family>.AppleSystemUIFont</family>
         </font>
        </property>
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buildButton">
        <property name="text">
         <string>4</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopButton">
        <property name="text">
         <string>T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="runButton">
        <property name="text">
         <string>5</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="submitButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="graphButton">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="terminalButton">
        <property name="text">
         <string>T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="dbButton">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="usesScrollButtons">
           <bool>true</bool>
          </property>
          <property name="documentMode">
           <bool>false</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="TaskBrowser" name="taskBrowser">
           <attribute name="title">
            <string>Task Browser</string>
           </attribute>
          </widget>
          <widget class="TestBrowser" name="testBrowser">
           <attribute name="title">
            <string>Test Cases</string>
           </attribute>
          </widget>
          <widget class="FileBrowser" name="fileBrowser">
           <attribute name="title">
            <string>File Browser</string>
           </attribute>
          </widget>
          <widget class="VerdictBrowser" name="verdictBrowser">
           <attribute name="title">
            <string>Verdict Table</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="TerminalWidget" name="terminal"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TerminalWidget</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.terminal.qterminal.widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VerdictBrowser</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.verdictbrowser.verdictbrowser</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileBrowser</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.filebrowser</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaskBrowser</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.taskbrowser</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TestBrowser</class>
   <extends>QWidget</extends>
   <header location="global">cchelper.testbrowser</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
